Volume/Views/MainView/HomeList.swift

```diff
@@ -101,19 +102,23 @@ struct HomeList: View {
                             }
                         }
                     }
-                    .padding([.bottom, .leading, .trailing])
+                    .padding([.leading, .trailing])
 
-                    Header("Following").padding(.bottom, -12)
+                    Header("Following", sidesWithDefaultPadding: [.leading, .trailing])
```

Comment:
Sounds good.  The reason I added that field was because there were so many places in the code with a Header that required default padding.  Adding that `sidesWithDefaultPadding` allowed me to edit the Headers that needed to have their padding adjusted, but kept all the other Headers the same since the default value of it is `[.top, .trailing, .bottom, .leading]`.  However, I can get rid of this and just manually adjust padding within code.